app/views/setup/discovery.html.slim

```diff
@@ -0,0 +1,34 @@
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
```

Comment:
Bootstrap Alerts won't be dismissed if its javascript components are not loaded.
Please double check that, I realised I didn't include them in my original PR so the functionality was broken.